### DISCUSSION TITLE
那位大佬能教我编译一下源码，可以有偿寻求援助

### DISCUSSION THREAD

**Discussion by v01d3x** [ttttupup/wxhelper#211] [General]
win11 64系统下编译

**Comment by vjeson** [comment]
我刚用visual studio ,编译成功，简单的可以教你一下 

cmake -DCMAKE_C_COMPILER=cl.exe  -DCMAKE_CXX_COMPILER=cl.exe -DCMAKE_ASM_MASM_COMPILER=ml64.exe -DCMAKE_BUILD_TYPE=Debug -DCMAKE_TOOLCHAIN_FILE:FILEPATH=D:\cpp\vcpkg\scripts\buildsystems\vcpkg.cmake -SE:\cpp\wxhelper 
路径换成你自己的本地路径，要先保证vcpkg安装正确 ，然后会生成vs project的sln项目文件,vs打开，编译会报错

主要spdlog应用编译不对，

修改vs的直接改成头文件引用，可以编译过，但是我一直注入不成功
sdplog引用修改方法
![image](https://github.com/ttttupup/wxhelper/assets/2101082/7f210fef-ec26-4600-a82c-a502e773cc72)

**Comment by isozone** [comment]
在cmake.txt中设置相关的路径，就可以编译成功了。
set(DETOURS_INCLUDE_DIRS "D:\\codes\\vcpkg\\installed\\x64-windows\\include")
set(DETOURS_LIBRARY "D:/codes/vcpkg/installed/x64-windows/debug/lib/detours.lib")
set(CMAKE_PREFIX_PATH "D:\\codes\\vcpkg\\installed\\x64-windows\\share\\nlohmann_json")

**Comment by liupeng176** [comment]
编译教程见：  https://github.com/ttttupup/wxhelper/issues/316
